outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "name" {
```

Comment:
Please add `fqdn` as well.
In the case of creating a domain, it is identical `name`.

This contract exists to be consistent and interchangeable with subdomains where `name` is different from `fqdn`.
Example subdomain `name = "api"`, `domain_name = "nullstone.io"`, `fqdn = "api.nullstone.io"`.